### PR TITLE
feat: start with multiple tabs with different paths

### LINKED ISSUE #1322: Start with multiple tabs with different paths
### Please describe the problem you're trying to solve

yazi only works with 1 path.
i dont see any options in the help pages that allows to open multiple tabs on startup.

eg.
`yazi ~/Downloads`

### Would you be willing to contribute this feature?

- [ ] Yes, I'll give it a shot

### Describe the solution you'd like

add option to start yazi with multiple tabs with different paths on startup

`yazi ~/Download ~/Documents ~/Public ~/Music`

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by coolkiid**
1. CLI can receive more than one paths as entries now.
2. Modify BOOT in yazi-boot/ and Tabs in yazi-core to fit the change above.
3. Add a test for parse_entries().

Closes #1322

For example, to open multi tabs at once, run `yazi ~/Projects ~/Downloads ~/Documents`.

### CODE REVIEW

File: yazi-boot/src/args.rs [hunk 1]
```diff
@@ -6,8 +6,8 @@ use clap::{command, Parser};
 #[command(name = "yazi")]
 pub struct Args {
 	/// Set the current working entry
-	#[arg(index = 1)]
-	pub entry: Option<PathBuf>,
+	#[arg(index = 1, value_delimiter = ' ', num_args = 1..)]
```
Comment (sxyazi) [inline yazi-boot/src/args.rs:9]: Please limit the maximum number of parameters to 9 as Yazi can only create up to 9 tabs.

Comment (coolkiid) [reply yazi-boot/src/args.rs]: done.

File: yazi-boot/src/args.rs [hunk 2]
```diff
@@ -6,8 +6,8 @@ use clap::{command, Parser};
 #[command(name = "yazi")]
 pub struct Args {
 	/// Set the current working entry
-	#[arg(index = 1)]
-	pub entry: Option<PathBuf>,
+	#[arg(index = 1, value_delimiter = ' ', num_args = 1..=9)]
+	pub entries: Vec<PathBuf>,
```
Comment (sxyazi) [inline yazi-boot/src/args.rs:10]: Why do we need `value_delimiter = ' '` here?

Comment (coolkiid) [reply yazi-boot/src/args.rs:10]: yea actually it is not necessary, i think it can be deleted. the behavior is the same with or without it.

**Review by sxyazi — APPROVED** [review]
LGTM, thank you so much for contributing this feature!

**Pull request merged**